radis/lbl/calc.py

```diff
@@ -267,8 +274,8 @@ def calc_spectrum(wavenum_min=None,
     :class:`~radis.lbl.factory.SpectrumFactory`, 
     the :ref:`Spectrum page <label_spectrum>`
     '''
-    
-    if not isinstance(molecule,dict):
+
+    if type(molecule) != list:
         s = _calc_spectrum(wavenum_min=wavenum_min,
```

Comment:
Here I would do : 
```suggestion
    if type(molecule) != list:
        molecule=[molecule]
```
And fall back to the other case. So you don't have to write s = _calc_spectrum(...) twice.